www/downloads.md

```diff
@@ -46,7 +46,12 @@ with elevated privileges:
 
 <li><strong>on Windows</strong>, by start a command prompt with <a
   href="https://technet.microsoft.com/en-us/library/cc947813.aspx">
-  administrator privileges</a> and launch the installer;</li>
+  administrator privileges</a> and launch the installer. Remember that these two environment variables
```

Comment:
That information is valid under both OSes (Windows and Linux), right?
What about moving the paragraph under the list?
We should also add a reference on how to set those variables under the two kind of OSes (e.g. https://www.java.com/en/download/help/path.xml).
I would also specify that PATH should either already point to the folder where the Jolie launchers (IIRC, that's how they are called in the Jolie installer) are installed or the user must add the Jolie launcher installation path into the PATH variable;

"jolie libs" -> "Jolie installation folder"